### ISSUE TITLE
Adding a comment to a node key not working properly

### ISSUE THREAD

**Issue by bloodmc** [zml2008/configurate#24]
I am unable to do the following 

```
# some comment
entity-activation-range {
   enabled=true
}
```

If I attempt to generate that, I end up with
```
entity-activation-range=null
```

However, setting comments to entries within a node is fine. For example, this works fine

```
entity-activation-range {
   # some comment
   enabled=true
}
```

**Comment by zml2008** [comment]
What code are you using to generate this? I can't repro in my tests.

**Comment by zml2008** [comment]
Fixed in a5dd815
